packages/plugins/rubygem-infoblox/rubygem-infoblox.spec

```diff
@@ -1,76 +1,85 @@
-# Generated from infoblox-1.0.0.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name infoblox
 
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 3.0.0
-Release: 1%{?dist}
-Summary: A Ruby wrapper to the Infoblox WAPI
+Release: 2%{?dist}
+Summary: This gem is a Ruby interface to the Infoblox WAPI. Using the gem, you can query, create, update, and delete DNS records in your Infoblox instance
 Group: Development/Languages
 License: MIT
 URL: https://github.com/govdelivery/infoblox
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
-%if 0%{?rhel} == 6
-Requires: ruby(abi)
-%else
-Requires: ruby(release)
-%endif
-Requires: ruby
-Requires: ruby(rubygems)
-Requires: rubygem(faraday)
-Requires: rubygem(faraday_middleware)
-%if 0%{?rhel} == 6
-BuildRequires: ruby(abi)
-%else
-BuildRequires: ruby(release)
-%endif
-BuildRequires: ruby
-BuildRequires: rubygems-devel
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(faraday)
+Requires: %{?scl_prefix}rubygem(faraday_middleware)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildArch: noarch
-Provides: rubygem(%{gem_name}) = %{version}
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end specfile generated dependencies
 
 %description
-This gem is a Ruby interface to the Infoblox WAPI. Using the gem, you can query,
-create, update, and delete DNS records in your Infoblox instance.
+A Ruby wrapper to the Infoblox WAPI.
+
 
 %package doc
-Summary: Documentation for %{name}
+Summary: Documentation for %{pkg_name}
 Group: Documentation
-Requires: %{name} = %{version}-%{release}
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
 BuildArch: noarch
 
 %description doc
-Documentation for %{name}.
+Documentation for %{pkg_name}.
 
 %prep
+%{?scl:scl enable %{scl} - << \EOF}
 gem unpack %{SOURCE0}
+%{?scl:EOF}
 
 %setup -q -D -T -n  %{gem_name}-%{version}
 
+%{?scl:scl enable %{scl} - << \EOF}
 gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
 
 %build
 # Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
 gem build %{gem_name}.gemspec
+%{?scl:EOF}
 
 # %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
 # by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
 %gem_install
+%{?scl:EOF}
 
 %install
 mkdir -p %{buildroot}%{gem_dir}
-cp -a .%{gem_dir}/* \
+cp -pa .%{gem_dir}/* \
```

Comment:
we dropped the `-p` in the templates in https://github.com/theforeman/foreman-packaging/commit/f02a98ade2c230f3968cd12e0ac220262f10109c